.travis.yml

```diff
@@ -8,19 +8,18 @@ cache:
 install:
   - pip3 install -r requirements.txt
 
-script:
-  - py.test -v --color=yes --cov=.
+jobs:
+  include:
+    - stage: test
```

Comment:
We can have a compile stage with the following command.
```
python -m compileall lib/
```
If the lib/deprecated does not compile, then you can skip that directory and add the rest. For more details on compileall module, please see [python docs](https://docs.python.org/3/library/compileall.html#module-compileall)

Reply:
It does compile the lib/deprecated